lib/rules/jsx-uses-vars.js

```diff
@@ -37,7 +37,7 @@ SOFTWARE.
 module.exports = {
   meta: {
     docs: {
-      description: 'Prevent variables used in JSX to be marked as unused',
+      description: 'prevent variables used in JSX to be marked as unused.', // eslint-disable-line consistent-docs-description
```

Comment:
Because it does not fit to descriptions which start with one of `enforce`, `require` and `disallow`. The `consistent-docs-description` rule which is cloned from `eslint` repo enforces the first word of the description.